packages/nodejs/package-lock.json

```diff
@@ -0,0 +1,2557 @@
+{
```

Comment:
This file shouldn't be committed I think, or created event. The workspace setup merges all lock files into the root lock file.